static/js/components/TelescopePage.jsx

```diff
@@ -70,7 +89,7 @@ const TelescopeList = ({ telescopes }) => {
   return (
     <div className={classes.root}>
       <List component="nav">
-        {telescopes?.map((telescope) => (
```

Comment:
Is the ? not a good thing?

Reply:
i actually didn't know for sure what it does. I'll put it back if you think that it is better.

Reply:
Yeah ? basically won't loop if telescopes is null, in case the variable is reloading and such.